### PR TITLE
PATCH: update the fhir get docs with defaults

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#799

Ticket: 799

### Description

Issue where the FHIR docs werent getting queried because we didnt pass doc ids

### Release Plan

- :warning: Points to `master`
- [ ] Merge this

### CODE REVIEW

**Review by leite08 — CHANGES_REQUESTED** [review]

File: packages/api/src/external/fhir/document/get-documents.ts [hunk 1]
```diff
@@ -23,14 +23,31 @@ export async function getDocuments({
...
-      const filtersAsStr = getFilters({ patientId, documentIds: docIds, from, to });
+    const defaultFilters = {
+      patientId,
+      from,
+      to,
+      documentIds,
+    };
+
+    if (documentIds && documentIds.length > 1) {
+      const chunksDocIds = chunk(documentIds, 150);
```
[the remark this answers is not in the document]
Comment (Orta21) [reply packages/api/src/external/fhir/document/get-documents.ts]: Yes it was got an error like 10 minutes ago in regards to URL being to large so im lowering to 150 now

[the remark this answers is not in the document]
Comment (leite08) [reply packages/api/src/external/fhir/document/get-documents.ts]: Sounds good!

As suggestion/good practice, it's good to add comments to these changes that others won't have a context as to why something is being done.

File: packages/api/src/command/medical/document/document-query.ts [hunk 2]
```diff
@@ -73,6 +74,12 @@ export async function queryDocumentsAcrossHIEs({
     cxDocumentRequestMetadata,
   });
 
+  await appendDocQueryProgressWithSource({
+    patient: updatedPatient,
+    requestId,
+    reset: true,
+  });
+
   getDocumentsFromCW({
```
Comment (leite08) [inline packages/api/src/command/medical/document/document-query.ts:83]: Downstream of `getDocumentsFromCW()` we also initialize the patient's DQ status, on `initPatientDocQuery()`.

Not only it would be good to have a single doc query initialization, but also it might be that the CW initialization code messes up with this one.

I'd consider removing the CW one and having only one DQ init code. But it needs to be tested, so it might be good to move this to its own PR.
